modules/auxiliary/gather/censys_search.rb

```diff
@@ -68,17 +74,14 @@ def search(keyword, search_type)
     end
 
     records = ActiveSupport::JSON.decode(res.body)
-    results = records['results']
+    results = records['result']
 
     if @searchtype.include?('certificates')
       parse_certificates(results)
     elsif @searchtype.include?('ipv4')
       parse_ipv4(results)
-    elsif @searchtype.include?('websites')
```

Comment:
Would it be possible to find the corresponding query with v2? I don't think we should remove the `website` enumeration from this module.